### PR TITLE
corrects plural name in models and prop name

### PR DESCRIPTION
**Pull request by invincibleJai**
corrects plural name in models and prop name for apiserver / kafka source

Tracks: https://jira.coreos.com/browse/ODC-2306

### CODE REVIEW

File: frontend/packages/knative-plugin/src/models.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const EventSourceApiServerModel: K8sKind = {
   kind: 'ApiServerSource',
   label: 'ApiServerSource',
   labelPlural: 'ApiServerSources',
-  plural: 'ApiServerSources',
+  plural: 'apiserversources',
```
Comment (andrewballantyne) [inline frontend/packages/knative-plugin/src/models.ts]: Are you sure about this change? This feels like it doesn't follow standard API conventions...

eg
```
export const PipelineRunModel: K8sKind = {
  apiGroup: 'tekton.dev',
  apiVersion: 'v1alpha1',
  label: 'Pipeline Run',
  plural: 'pipelineruns', // <<<<<<
  abbr: 'PLR',
  namespaced: true,
  kind: 'PipelineRun',
  id: 'pipelinerun',
  labelPlural: 'Pipeline Runs',
  crd: true,
  badge: BadgeType.DEV,
};
```
`plural` is an API-level name, not a readable name. So it's "camelCase".

Comment (andrewballantyne) [reply frontend/packages/knative-plugin/src/models.ts]: Hah, apparently I can't read... you are putting it to lowercase, not the other way around.

**Comment by karthikjeeyar** [comment]
kafka-source
@invincibleJai Verified locally, It works fine.

**Comment by andrewballantyne** [comment]
Wow, nice... frontend tests hit OOM:

> could not wait for build: the build test-bin failed after 8m27s with reason OutOfMemoryKilled: The build pod was killed due to an out of memory condition.

File: frontend/packages/knative-plugin/src/models.ts (hunk 1 above)
Comment (invincibleJai) [reply frontend/packages/knative-plugin/src/models.ts]: yeah `plural`  is used to fetch resources so all models has lowercase

**Pull request merged**